beeline-examples/package.yaml

```diff
@@ -48,6 +48,11 @@ when:
       ghc-options:
         - -Wall
 
+  - condition: flag(strict) && impl(ghc >= 9.8.1)
```

Comment:
Oh interesting. I would have thought to nest this inside the the strict condition. But nothing _wrong_ with doing it here.